Yoast/wordpress-seo
# Add endpoint for image size

## Summary

This PR can be summarized in the following changelog entry:

* Introduces an endpoint for getting an attachment file size.

## Test instructions

This PR can be tested by following these steps:
For testing the endpoint you have to use a nonce. This is a 'secret' string that should be send within your request. To get this nonce you can go to the WordPress admin dashboard and in your console you can do: `wpApiSettings.nonce`. If everything works well you get the nonce.

To use the nonce you have to append this to the request url: `url?_wpnonce={the nonce}`. This means in this case: `your-test-domain/wp-json/yoast/v1/file_size?url={an-url-to-a-attachment-file}&_wpnonce={the nonce}`

* Get a url of an image. This should be uploaded locally to the media library
* Checkout this branch.
* In your browser open the endpoint by `your-test-domain/wp-json/yoast/v1/file_size?url={an-url-to-a-attachment-file}`

## Quality assurance
